lib/guardian/token/jwt.ex

```diff
@@ -359,29 +360,23 @@ defmodule Guardian.Token.Jwt do
     do: assign_exp_from_ttl(the_claims, {iat_v, requested_ttl})
 
   # catch all for when the issued at iat is not yet set
-  defp set_ttl(claims, requested_ttl) do
-    claims |> set_iat() |> set_ttl(requested_ttl)
-  end
+  defp set_ttl(claims, requested_ttl),
+    do: claims |> set_iat() |> set_ttl(requested_ttl)
 
-  defp assign_exp_from_ttl(the_claims, {iat_v, {seconds, unit}}) when unit in [:second, :seconds] do
-    Map.put(the_claims, "exp", iat_v + seconds)
-  end
+  defp assign_exp_from_ttl(the_claims, {iat_v, {seconds, unit}}) when unit in [:second, :seconds],
```

Comment:
Made these inline functions to be consistent with the rest of the file's single line function definitions.